doc/api/process.md

```diff
@@ -373,7 +373,10 @@ It is important to take note of the following:
 * `SIGTERM` is not supported on Windows, it can be listened on.
 * `SIGINT` from the terminal is supported on all platforms, and can usually be
   generated with `CTRL+C` (though this may be configurable). It is not generated
-  when terminal raw mode is enabled.
+  when terminal raw mode is enabled. If your program listens for `SIGINT`, it is
```

Comment:
it might be good to be explicit that you would be listening on process.on('SIGINT')... it may also be redundant. Thoughts?